juju-charms/charm-trilio-data-mover-api/src/reactive/dmapi_handlers.py

```diff
@@ -169,5 +165,4 @@ def cluster_connected(hacluster):
 
 @reactive.hook('upgrade-charm')
 def upgrade_charm():
-    # Remove sate
-    reactive.remove_state('charm.installed')
+    dmapi.install()
```

Comment:
Need to clear the flag for charms upgrade, dmapi.install() is not called if flag is not cleared

Reply:
will update the code accordingly.